app/views/solution-card.test.js

```diff
@@ -112,7 +133,30 @@ describe('solution-card', () => {
         done();
       });
   });
+  {/* <div data-test-id="solution-card-name" class="nhsuk-u-margin-top-5">
+<a href="/view-solution/{{ solution.id }}">View the solution</a>
+</div> */}
+  it('should render the View the solution link', (done) => {
```

Comment:
Do we need this comment?

Reply:
haha no it was for my reference :D I will remove